go/private/context.bzl

```diff
@@ -578,14 +570,9 @@ def go_context(ctx, attr = None):
         root = goroot,
         go = binary,
         stdlib = stdlib,
```

Comment:
Yay for making this smaller. It's technically public API, but I don't want to view it as such.